README.md

```diff
@@ -139,9 +156,8 @@ POST    /file-routing/requests
 | --------|---------|-------|
 | Created  | 201   | Successfully created event CLOSED |
 | Bad Request  | 400   |  Invalid Request. |
```

Comment:
400 = invalid request, this description doesn't add value, explaining when we may return 400 does